src/cubic_loader/qlik/ods_tables.py

```diff
@@ -20,7 +20,7 @@
     "EDW.TRIP_PAYMENT",
     "EDW.SALE_TRANSACTION",
     "EDW.PAYMENT_TYPE_DIMENSION",
-    # "EDW.TRANSACTION_HISTORY", # Removed for load failure OOM issue 7/11/25
+    "EDW.TRANSACTION_HISTORY",
```

Comment:
PR #67 changed the memory usage profile for the CubicODSQlik process. ECS memory usage returns to almost 0% between job runs, this SHOULD allow the TRANSACTION_HISTORY table to compete. If not, PR #67 also protects a failure in one table from blocking the loading of all other tables.